<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddServerWidget</class>
 <widget class="QWidget" name="AddServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddServerWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="_24">
        <item>
         <layout class="QVBoxLayout" name="_25">
          <item>
           <widget class="QLabel" name="textLabel1_4">
            <property name="text">
             <string>Display Name:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="m_addrLabel_2">
            <property name="text">
             <string>Host Name or IP:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="_26">
          <item>
           <widget class="QLineEdit" name="m_nameEdit">
            <property name="toolTip">
             <string>The name that this server is displayed as.</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="_27">
            <item>
             <widget class="QLineEdit" name="m_addrEdit">
              <property name="toolTip">
               <string>For SSL connections use hostname otherwise use hostname or IP address</string>
              </property>
              <property name="maxLength">
               <number>512</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="m_portLabel_2">
              <property name="text">
               <string>Port:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_portEdit">
              <property name="text">
               <string>119</string>
              </property>
              <property name="maxLength">
               <number>12</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="pausedCheckBox">
          <property name="text">
           <string>Paused</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_28">
        <item>
         <layout class="QHBoxLayout" name="_29">
          <item>
           <widget class="QLabel" name="textLabel7_2">
            <property name="text">
             <string>Priority:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSpinBox" name="m_priorityInput">
          <property name="toolTip">
           <string>Load balancing is carried out based on the server priority, type and current server queue size.</string>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer5_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>18</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_30">
          <item>
           <widget class="QLabel" name="textLabel8_2">
            <property name="text">
             <string>Maximum Threads:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSpinBox" name="m_threadInput">
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer4_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>29</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_31">
          <item>
           <widget class="QLabel" name="textLabel1_5">
            <property name="text">
             <string>Retries:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="m_retriesInput">
            <property name="toolTip">
             <string>The number of times a download is retried following a fatal error</string>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_32">
        <item>
         <layout class="QHBoxLayout" name="_33">
          <item>
           <widget class="QLabel" name="textLabel6_2">
            <property name="text">
             <string>Server connection timeout (secs):</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSpinBox" name="m_timeoutInput">
          <property name="toolTip">
           <string>Time to wait if the server stops responding before dropping connection and reconnecting</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>600</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer6_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>31</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_34">
          <item>
           <widget class="QLabel" name="textLabel1_6">
            <property name="text">
             <string>Thread timeout (Minutes):</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="m_threadTimeoutInput">
            <property name="maximum">
             <number>2147483647</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_19">
        <item>
         <widget class="QCheckBox" name="m_logCheck">
          <property name="toolTip">
           <string>Do you need to provide a username and password?</string>
          </property>
          <property name="text">
           <string>Server requires authentication</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="serverTypeLabel_2">
          <property name="text">
           <string>Server Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="serverType">
          <property name="maxVisibleItems">
           <number>5</number>
          </property>
          <item>
           <property name="text">
            <string>Active</string>
           </property>
           <property name="icon">
            <iconset resource="quban.qrc">
             <normaloff>:/quban/images/ginux/Entire_Network-64.png</normaloff>:/quban/images/ginux/Entire_Network-64.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Passive</string>
           </property>
           <property name="icon">
            <iconset resource="quban.qrc">
             <normaloff>:/quban/images/ginux/Passive_Network-64.png</normaloff>:/quban/images/ginux/Passive_Network-64.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dormant</string>
           </property>
           <property name="icon">
            <iconset resource="quban.qrc">
             <normaloff>:/quban/images/ginux/Dormant_Network-64.png</normaloff>:/quban/images/ginux/Dormant_Network-64.png</iconset>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="sslCheck">
          <property name="text">
           <string>SSL connection</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="_21">
          <item>
           <widget class="QLabel" name="textLabel5_2">
            <property name="text">
             <string>User name:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel4_2">
            <property name="text">
             <string>Password:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="_22">
          <item>
           <widget class="QLineEdit" name="m_userEdit">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_passEdit">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="limitsCheckBox">
            <property name="text">
             <string>Enable Server limits</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="limitsSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="limitsUnitsComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="limitsFreqComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="currentDownloadsLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Current Downloads:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="currentDownloadsSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="resetLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Period Start:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="resetDateEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Server limits start : cannot be in the future&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="resetEndLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Period End:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="resetEndDateEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Use these Server Extensions</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QListWidget" name="serverExtensionsList">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Ignore these SSL Connection Errors</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QListWidget" name="SSLErrorsList">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_18">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <spacer name="Horizontal Spacing2_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonOk">
          <property name="text">
           <string>&amp;OK</string>
          </property>
          <property name="shortcut">
           <string/>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updateButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="testServerButton">
          <property name="toolTip">
           <string>Tests whether you can connect to the server: does not test authentication</string>
          </property>
          <property name="text">
           <string>Test Server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonCancel">
          <property name="text">
           <string>&amp;Cancel</string>
          </property>
          <property name="shortcut">
           <string/>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="quban.qrc"/>
 </resources>
 <connections/>
</ui>
